### PR TITLE
Fix workflow execution

### PR DESCRIPTION
**Pull request by Alan5142**

### CODE REVIEW

File: src/engine_core/src/input/InputManager.hpp [hunk 1]
```diff
@@ -19,5 +19,5 @@ class InputManager
 
   private:
     // TODO: Reserve memory for this map???
-    static std::map<KeyCode, KeyData> s_keyDataByCode;
-};
\ No newline at end of file
+    static std::map<KeyCode, KeyData> S_KEY_DATA_BY_CODE;
```
Comment (Kyn21kx) [inline src/engine_core/src/input/InputManager.hpp:22]: Is this naming convention the standard for static properties?
If so, that's cool, just wondering

File: src/engine_core/src/input/KeyCode.hpp [hunk 2]
```diff
@@ -141,22 +141,22 @@ enum class EKeyCode : KeyCode
...
-    KP_5 = 93,
-    KP_6 = 94,
-    KP_7 = 95,
-    KP_8 = 96,
-    KP_9 = 97,
-    KP_0 = 98,
-    KP_PERIOD = 99,
+    KpDivide = 84,
+    KpMultiply = 85,
+    KpMinus = 86,
```
Comment (Kyn21kx) [inline src/engine_core/src/input/KeyCode.hpp:146]: Thanks for renaming these 🙏🏻

**Review by Kyn21kx — APPROVED** [review]
Just a small question

File: src/engine_core/src/input/InputManager.hpp (hunk 1 above)
Comment (Alan5142) [reply src/engine_core/src/input/InputManager.hpp:22]: That's the specified one in .clang-tidy, we can change it. :)

File: src/engine_core/src/input/KeyCode.hpp (hunk 2 above)
Comment (Alan5142) [reply src/engine_core/src/input/KeyCode.hpp:146]: I would like to keep it consistent, but clang-tidy kept the SCREAMING on most of them

**Pull request merged**